spec/javascripts/controllers/containers/container_topology_controller_spec.js

```diff
@@ -27,4 +27,11 @@ describe('containerTopologyController', function() {
             expect($scope.kinds).not.toBe(undefined);
         });
     });
+
+    describe('kinds contain all expected kinds', function() {
+        it('in all main objects', function() {
+               expect(Object.keys($scope.kinds).length).toBeGreaterThan(7);
```

Comment:
This is indented 8 spaces more than the `it`. Even though the file is using 4 spaces, not 8.
But the closing `});` both have only 4 spaces less than this line .. so it's broken.

We do have a style of 2 spaces, and if you want to convert the whole file, I'm all for it, but right now, I'd settle for consistency within the file at least.